ckanext/versions/cli.py

```diff
@@ -38,5 +46,62 @@ def cleandb(ctx):
     click.secho('Dataset versions tables created', fg="green")
 
 
+@versions.command()
+@click.pass_context
+@click.option('--name', default='1.0')
+@click.option('--notes', default='Initial version.')
+def create_initial_version(ctx, name, notes):
+    """Creates an initial resource version for all existing resources.
+
+    The creator of the version will be set as the package creator.
+    """
+    resources = _get_list_of_resources_without_version()
+
+    if not resources:
+        click.secho('All resources contains at least one version.', fg="green")
+        ctx.exit(1)
+
+    click.secho(
+        "Creating initial version for {} resources.".format(len(resources))
+        )
+    for resource in resources:
+        _create_resource_initial_version(resource, name, notes)
+    click.secho("Initial versions created succesfully.")
+
+
+def _get_list_of_resources_without_version():
+    session = Session()
+    resources = session.query(Resource.id, Package.creator_user_id).\
+        filter(Resource.state != 'deleted').\
+        join(Package).\
+        order_by(Resource.created).\
+        all()
+
+    if not resources:
+        return None
+
+    versions = session.query(Version.resource_id).distinct()
+
+    resources_with_version = [v.resource_id for v in versions]
+
+    result = [r for r in resources if r.id not in resources_with_version]
+
+    return result
+
+
+def _create_resource_initial_version(resource, name, notes):
+    context = {'ignore_auth': True}
+    data_dict = {
+        'resource_id': resource.id,
+        'name': name,
+        'notes': notes,
+        'creator_user_id': resource.creator_user_id
```

Comment:
Does `resource.creator_user_id` exist? My understanding is that it's a dataset field